assets/pages/bundles-landing/reducers/reducers.js

```diff
@@ -5,6 +5,7 @@
 import { combineReducers } from 'redux';
 
 import { abTestReducer as abTests } from 'helpers/abtest';
+import { intCmpReducer as intCmp } from 'helpers/intCmp';
```

Comment:
Just a question about style:

You could make intCmpReducer the default export and then here just put:

import intCmp from 'helpers/intCmp';

Is there a reason to prefer the way you've done it? Should we care about standardising stuff like this? @Ap0c What do you think?